packages/zowe-explorer-api/src/tree/IZoweTreeNode.ts

```diff
@@ -181,6 +181,11 @@ export interface IZoweDatasetTreeNode extends IZoweTreeNode {
      */
     encoding?: string;
 
+    /**
+     * Use Dataset-specific tree node for children.
+     */
+    children?: IZoweDatasetTreeNode[];
```

Comment:
should we deprecate `getChildren()` of do we still need an async approach to get them? 😋 

Same for USS

Reply:
I believe we still need an async approach to fetch items from the mainframe when we want to update the tree 😋